docs/webhooks/index.html

```diff
@@ -625,6 +626,36 @@ <h4 id="companyTestSessionFinished">companyTestSessionFinished</h4>
     plagiarismLabel: 'medium',
     url: https://app.codesignal.com/test-result/oH3qeBC38oFsf7qB4?accessToken=A7HnBpab4aD7xm7Kp-iPhk2se5Wnxeg7x9GruANLzn
   }
+};</code></pre>
+            </div>
+            <div class="subsection">
+              <h4 id="preScreenIntegrityReviewUpdated">preScreenIntegrityReviewUpdated</h4>
+              <p>Fired when a non-proctored test session's integrity review status is updated.</p>
+              <div>Request format:</div>
+              <pre><code>{
+  event: 'preScreenIntegrityReviewUpdated',
+  triggeredOn: number,
+  payload: {
+    testSessionId: string,
+    testId: string,
+    testTitle: string,
+    candidateEmail: string,
+    candidateName: string,
+    integrityReviewSuggested: boolean | null,
```

Comment:
Following the discussion in the main pull request we might want to update this type to be just `boolean`.